src/geometries/ExtrudeGeometry.js

```diff
@@ -439,6 +439,24 @@ class ExtrudeGeometry extends BufferGeometry {
 
 			}
 
+			//When bevelSegments == 0, the modified vert arrays will not have been populated. We do this here.
+
+			if ( contractedContourVertices.length == 0 ) {
+
+				contractedContourVertices.push( ...contour );
+
+			}
+
+			if ( expandedHoleVertices.length == 0 ) {
+
+				for ( let h = 0, hl = numHoles; h < hl; h ++ ) {
+
+					expandedHoleVertices.push( holes[ h ] ); //triangulateShape expects a Vec2[][], ie. an array of holes, with each hole being a Vec2[]
+
+				}
+
+			}
```

Comment:
Is there ever a case where contract contour will have length 0 but expandedHoleVertices will not? I think for clarity we should handle these in the same condition.